.travis.yml

```diff
@@ -34,29 +39,56 @@ install:
 
 script:
   - source "$DEV_LIB_PATH/travis.script.sh"
+  - if [ ! -z $JS_TESTS ]; then npm run test:js; fi
 
 after_script:
   - source "$DEV_LIB_PATH/travis.after_script.sh"
 
 jobs:
+  fast_finish: true
   include:
+    - stage: lint
+      name: Lint
+      php: "7.3"
+      env: WP_VERSION=latest DEV_LIB_SKIP=phpunit
+
     - stage: test
+      name: JavaScript unit tests
       php: "7.3"
-      env: WP_VERSION=latest
-    - php: "5.4"
-      env: WP_VERSION=4.9    DEV_LIB_SKIP=phpcs
-    - php: "5.5"
-      env: WP_VERSION=5.0    DEV_LIB_SKIP=phpcs
-    - php: "5.6"
-      env: WP_VERSION=5.1    DEV_LIB_SKIP=phpcs
-    - php: "5.6"
-      env: WP_VERSION=latest DEV_LIB_SKIP=phpcs
-    - php: "7.0"
-      env: WP_VERSION=latest DEV_LIB_SKIP=phpcs
-    - php: "7.1"
-      env: WP_VERSION=latest DEV_LIB_SKIP=phpcs
-    - php: "7.2"
-      env: WP_VERSION=latest DEV_LIB_SKIP=phpcs
+      env: WP_VERSION=latest DEV_LIB_SKIP=phpcs,eslint,xmllint,phpsyntax,phpunit JS_TESTS=1
+
+    - name: PHP unit tests (7.3, WordPress latest)
+      php: "7.3"
+      env: WP_VERSION=latest DEV_LIB_SKIP=phpcs,eslint,xmllint,phpsyntax
```

Comment:
Instead of `DEV_LIB_SKIP` you can instead use `DEV_LIB_ONLY` to just select the tests you want.